### ISSUE TITLE
[Accessibility]: cannot navigate to all buttons on Code Monitoring View page

### ISSUE THREAD

**Issue by camdencheek** [sourcegraph/sourcegraph#35140]
### Audit type

Keyboard navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34192

### Problem description

On Firefox v101.0b4, tab navigation through the "view code monitors" page does not iterate over all actionable buttons and links. This screenshot highlights the actions that cannot be navigated to with the keyboard. 

<img width="1512" alt="Screen Shot 2022-05-09 at 14 45 55" src="https://user-images.githubusercontent.com/12631702/167476984-90047175-a262-414b-9db5-b3983ed16176.png">

### Expected behavior

It should be possible to select highlighted buttons by cycling through fields with `Tab`.

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by GitStart. I have assigned the label 'gitstart' and added the assignee 'gitstart-sourcegraph'.

**Comment by gitstart-sourcegraph** [comment]
Hello @camdencheek 
Did a bit of research here and there, Found a related question on stackoverflow and [this answer](https://stackoverflow.com/a/11713537). The bug can be reproduced as described in this issue, however after implementing the stackoverflow answer, the issue was solved. WDYT?

**Comment by camdencheek** [comment]
Thanks for looking into it! I can confirm that the steps described in that StackOverflow answer fix the issue for me. I did not have to modify `about:config` for my combination of Firefox/macOS. 

cc @limitedmage I think we can close this, but want to confirm with someone who is a bit more familiar with the space
